tests/e2e/mock_servers/workday_routes.py

```
@@ -0,0 +1,96 @@
+
+@workday.route("/d/home.htmld")
+def home():
+    """Workday home page with Time button."""
+    return render_template("workday_home.html")
+
+
+@workday.route("/d/time.htmld")
+def time_page():
+    """Time page with week selection and calendar."""
+
+    global _current_week_start  # noqa: PLW0603
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `PLW0603`)

Remove unused `noqa` directive

(RUF100)